goal_src/jak1/engine/debug/default-menu.gc

```diff
@@ -4480,12 +4480,59 @@
       )
   )
 
+(defun dm-music-player-func ((lst object))
+  (let ((proc (process-by-name 'dm-player *active-pool*)))
+    (if proc
+        (kill-by-name 'dm-player *active-pool*)
+        (set! *progress-flava* -1)
+      )
+    )
```

Comment:
You can use the `aif` macro here and save yourself the let. You can also just omit the check entirely, it's fine for `kill-by-name` not to find anything.